leapp/cli/upgrade/__init__.py

```diff
@@ -209,13 +213,18 @@ def preupgrade(args):
 
     report_errors(workflow.errors)
 
-    report_txt, report_json = [os.path.join(get_config().get('report', 'dir'),
+    cfg = get_config()
+    report_txt, report_json = [os.path.join(cfg.get('report', 'dir'),
                                             'leapp-report.{}'.format(f)) for f in ['txt', 'json']]
-    report_info([report_txt, report_json], fail=workflow.errors)
     # fetch all report messages as a list of dicts
     messages = fetch_upgrade_report_raw(context, renderers=False)
     with open(report_json, 'w+') as f:
         json.dump({'entries': messages}, f, indent=2)
+    report_files = [os.path.join(cfg.get('report', 'dir'), r) for r in cfg.get('report', 'files').split(',')]
+    modified_report_files = [rf for rf in report_files
```

Comment:
Currently modified_report_files is [] and report_info doesn't report anything

Reply:
You'll need to reconsider timestamp comparison, the way it is now the list will remain empty:
```ipdb> [unix_timestamp_to_iso(os.path.getmtime(r)) for r in report_files]
['2019-08-08T16:22:01.372725Z', '2019-08-08T16:22:01.346725Z', '2019-08-08T16:22:01.364725Z']
ipdb> e.stamp
'2019-08-08T20:19:31.968004Z'
```

Reply:
Yeah, looks like a timezone issue. I'll try to dig into that, thanks.

Reply:
Okay, now it's always comparing UTC to UTC.
Tested on 3 VMs with timezone set to UTC, New York and Prague respectively - all OK.